sympy/physics/control/lti.py

```diff
@@ -0,0 +1,553 @@
+        >>> G1 = TransferFunction(p**2 + 2*p + 4, p - 6, p)
+        >>> G1.var
+        p
+        >>> G2 = TransferFunction(0, s - 5, s)
+        >>> G2.var
+        s
+
+        """
+        return self._var
+
+    @property
+    def bound_symbols(self):
+        """Returns only variables that are dummy variables.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, l
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction(s + l, s - 2*p, s)
+        >>> G1.bound_symbols
+        [s]
+        >>> G2 = TransferFunction(p**2 + 2*p + 4, p - 6, p)
+        >>> G2.bound_symbols
+        [p]
+
+        """
+        return [self.var]
+
+    def _eval_subs(self, old, new):
+        arg_num = self.args[0].subs(old, new)
+        arg_den = self.args[1].subs(old, new)
+        argnew = TransferFunction(arg_num, arg_den, self.var)
+        return self if old in self.bound_symbols else argnew
+
+    def _eval_evalf(self, prec):
+        return TransferFunction(
+            self.num._eval_evalf(prec),
+            self.den._eval_evalf(prec),
+            self.var)
+
+    def _eval_simplify(self, **kwargs):
+        tf = cancel(Mul(self.num, 1/self.den)).as_numer_denom()
+        num_, den_ = tf[0], tf[1]
+        return TransferFunction(num_, den_, self.var)
+
+    def expand(self):
+        """
+        Returns the Transfer Function with Numerator and Denominator
+        in expanded form.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction((a - s)**2, (s**2 + a)**2, s)
+        >>> G1.expand()
+        TransferFunction(a**2 - 2*a*s + s**2, a**2 + 2*a*s**2 + s**4, s)
+        >>> G2 = TransferFunction((p + 3*b)*(p - b), (p - b)*(p + 2*b), p)
+        >>> G2.expand()
+        TransferFunction(-3*b**2 + 2*b*p + p**2, -2*b**2 + b*p + p**2, p)
+
+        """
+        return TransferFunction(expand(self.num), expand(self.den), self.var)
+
+    def __add__(self, other):
+        """Adds Transfer Functions.
+
+        Parameters
+        ==========
+
+        other : TransferFunction
+            The Transfer Function to add to current (self) Transfer Function.
+
+        Returns
+        =======
+
+        TransferFunction
+            The resultant Transfer Function after adding (with '+' operator) self to other.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, a
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(s + 6, s - 5, s)
+        >>> tf2 = TransferFunction(s + 3, s + 1, s)
+        >>> tf1 + tf2
+        TransferFunction((s - 5)*(s + 3) + (s + 1)*(s + 6), (s - 5)*(s + 1), s)
+        >>> tf1 + (a - 1)
+        TransferFunction(s + (a - 1)*(s - 5) + 6, s - 5, s)
+        >>> tf1 + 8
+        TransferFunction(9*s - 34, s - 5, s)
+
+        """
```

Comment:
Dunder methods do not need docstrings in general, as users won't see them. Move these examples into the class level docstring and remove this docstring.